qtop.py

```diff
@@ -1727,14 +1643,15 @@ def decide_batch_system(cmdline_switch, env_var, config_file_batch_option):
                     add_to_sample(source_files, savepath, subdir='source')
 
                 if scheduler == "pbs":
-                    scheduling_system = PBSBatchSystem(in_out_filenames)
+                    scheduling_system = PBSBatchSystem(in_out_filenames, config)
```

Comment:
I have a strong feeling that `in_out_filenames` can be derived out of the `config`.... for future iteration.